### PR TITLE
Add discharge token for user endpoint.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/132/
Test PASSed.

File: theblues/identity_manager.py [hunk 1]
```diff
@@ -90,6 +90,23 @@ def discharge(self, username, macaroon):
         json_macaroon = json.dumps(macaroon)
         return base64.urlsafe_b64encode(json_macaroon.encode('utf-8'))
 
+    def discharge_token(self, username):
+        """Discharge thoken for a user.
```
Comment (bac) [inline theblues/identity_manager.py]: typo: token

**Comment by bac** [comment]
:+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/133/
Test PASSed.

File: theblues/identity_manager.py [hunk 2]
```diff
@@ -74,6 +74,8 @@ def discharge(self, username, macaroon):
         @param username The logged in user.
         @param macaroon The macaroon returned from the charm store.
         @return The resulting base64 encoded macaroon.
+        @raises ServerError when making request to the discharge endpoint
```
Comment (kadams54) [inline theblues/identity_manager.py:77]: I don't see a ServerError being raised anywhere? Probably just missing something.

**Comment by kadams54** [comment]
:+1: with one question. Thanks!

File: theblues/identity_manager.py (hunk 2 above)
Comment (fabricematrat) [inline theblues/identity_manager.py:77]: @kadams54 from make request

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/theblues-lib-merge

**Pull request merged**